### DISCUSSION TITLE
Iperf API for windows

### DISCUSSION THREAD

**Discussion by halian95** [esnet/iperf#1548] [Q&A]
I want to run network diagnostics with Iperf (client mode) as a part of a Windows executable software written in C++. Is there any usable Iperf API / package that I can use as the dependency for this means?

Currently, I am running Iperf as a subprocess inside the code and capturing standard output, which is a bit messy.

**Comment by swlars** [comment]
We don't support Windows, but we do have a library for UNIX/Linux-based systems. We're not sure about the broader community.
